packages/framework/presence/src/test/notificationsManager.spec.ts

```diff
@@ -264,12 +265,18 @@ describe("Presence", () => {
 				false,
 			);
 
-			assert(eventHandlerCalls.original.length === 1, "original event handler was not called");
+			assert(
+				eventHandlerCalls.original.length === 1,
+				`original event handler was called ${eventHandlerCalls.original.length} times; expected 1`,
+			);
```

Comment:
`assert.equal(eventHandlerCalls.original.length, 1` is preferred (easier for authors)